packages/api/src/services/erc20-state.ts

```diff
@@ -0,0 +1,40 @@
+import { clients } from "@uma/sdk";
+import { Libs, Json } from "..";
+import { asyncValues } from "../libs/utils";
+type Config = Json;
+export default function (config: Config, libs: Libs) {
+  const { provider, erc20s, collateralAddresses, syntheticAddresses } = libs;
+  async function getTokenState(address: string) {
+    const instance = clients.erc20.connect(address, provider);
+    return asyncValues({
```

Comment:
I'm assuming this is meant to only be used as an internal function (might be helpful to have a way to signal that)? if not, one might expect this to pull from the cache... but it seems this is meant to update the cache. An external caller might find this confusing since it _appears_ to be a getter.

Reply:
yes you are right, its meant to be internal, its technically is a getter though , it just returns the queried token state, the update happens in the `updateToken` function, which is also internal.

Reply:
i see what you are saying though, its ambiguous where the source of the "get" is, yea thats a good point